nvim-treesitter-scm-1.rockspec

```diff
@@ -0,0 +1,48 @@
+local MODREV, SPECREV = 'scm', '-1'
+rockspec_format = '3.0'
+package = 'nvim-treesitter'
+version = MODREV .. SPECREV
+
+description = {
+  summary = 'Nvim Treesitter configurations and abstraction layer',
+  labels = { 'neovim' },
+  homepage = 'https://github.com/nvim-treesitter/nvim-treesitter',
+  license = 'Apache-2.0',
+}
+
+dependencies = {
+  'lua >= 5.1',
+}
+
+source = {
+  url = 'http://github.com/nvim-treesitter/nvim-treesitter/archive/v' .. MODREV .. '.zip',
+  dir = '.'
+}
+
+if MODREV == 'scm' then
+  source = {
+    url = 'git://github.com/nvim-treesitter/nvim-treesitter',
+  }
+end
+
+build = {
+  type = 'make',
+}
+
+if MODREV == 'scm' then
+  build = {
+    type = 'make',
+    install_variables = {
+      INST_PREFIX='$(PREFIX)',
+      INST_BINDIR='$(BINDIR)',
+      INST_LIBDIR='$(LIBDIR)',
+      INST_LUADIR='$(LUADIR)',
+      INST_CONFDIR='$(CONFDIR)',
+    },
+    copy_directories = {
+      'autoload',
+      'plugin',
+      'queries'
+    }
+  }
+end
```

Comment:
I'm not sure if we need the build type to be `'make'` for `'scm'` versions.

Reply:
we need a Makefile because the default luarocks builder will try to build all the .c files it sees, except this fails because in the test folder, there are faulty .c files precisely to test treesitter on failure.